kubeless-rbac-novols.jsonnet

```diff
@@ -0,0 +1,9 @@
+# Remove volumeClaimTemplates from kafkaSts to enable testing kubeless
```

Comment:
hrm, it's a shame this is necessary.  Is there no PV dynamic provisioner that will work on kubeadm-dind-cluster?  (perhaps the new k8s 1.7 'dynamic' host-fs provisioner?)